### PR TITLE
Update get-set-up-for-aws-batch.md

### PR DESCRIPTION
**Pull request by landermkerbey**
Chiefly minor typo fixes. The change to the section about creating an IAM user was about enforcing subject-verb agreement. Depending on intent, it might make more sense to render everything singular, but based on how I read this document, plural made sense. Please advise.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by nrdlngr** [comment]
Thanks for your interest in contributing to the AWS Batch User Guide. Looks like that colon should be a semicolon; nice catch. 

I'm not sure I agree with your other rewording suggestions, but if you revert all lines except for the colon-semicolon change, I'll gladly merge this.

**Comment by landermkerbey** [comment]
@nrdlngr 

What about the 'or and' in the passage titled "Create an IAM User"? If the intent is to say doing both or doing either is appropriate, what about 'and/or'?

Also, what should be done about the subject-verb conflict in the statement 'You must create IAM _roles_ that _provides_ these credentials to your compute environments and container instances and then associate _that_ role with your compute environments'? Do you want that to be singular or plural?

I also noticed that the last paragraph includes the statement 'o use the AWS CLI with AWS Batch, install the latest AWS CLI, version.' Should that be changed to add a specific version number after version, or should I remove the comma?

File: doc_source/get-set-up-for-aws-batch.md [hunk 1]
```diff
@@ -95,7 +95,7 @@ For more information about IAM, see the [AWS Identity and Access Management User
 
 ## Create IAM Roles for your Compute Environments and Container Instances<a name="create-an-iam-role"></a>
 
-Your AWS Batch compute environments and container instances require AWS account credentials to make calls to other AWS APIs on your behalf\. You must create IAM roles that provides these credentials to your compute environments and container instances and then associate that role with your compute environments\.
+Your AWS Batch compute environments and container instances require AWS account credentials to make calls to other AWS APIs on your behalf\. You must create IAM roles that provide these credentials to your compute environments and container instances, then associate those roles with your compute environments\.
```
Comment (nrdlngr) [inline doc_source/get-set-up-for-aws-batch.md]: Sorry, I missed the nuance here on the first look. Apologies. Please make this "create an IAM role" "associate that role".

File: doc_source/get-set-up-for-aws-batch.md [hunk 2]
```diff
@@ -41,7 +41,7 @@ Note your AWS account number, because you'll need it for the next task\.
...
+Services in AWS, such as Amazon EC2 and AWS Batch, require that you provide credentials when you access them, so that the service can determine whether you have permission to access its resources\. The console requires your password\. You can create access keys for your AWS account to access the command line interface or API\. However, we don't recommend that you access AWS using the credentials for your AWS account; we recommend that you use AWS Identity and Access Management \(IAM\) instead\. Create an IAM user, and then add the user to an IAM group with administrative permissions or grant this user administrative permissions\. You can then access AWS using a special URL and the IAM user's credentials\.
```
Comment (nrdlngr) [inline doc_source/get-set-up-for-aws-batch.md:44]: I couldn't see the "and/or" change here before until you called it out in the conversation. You are correct, this should be "or".

File: doc_source/get-set-up-for-aws-batch.md [hunk 3]
```diff
@@ -205,4 +205,4 @@ For security reasons, we don't recommend that you allow SSH access from all IP a
 
 ## Install the AWS CLI<a name="install_aws_cli"></a>
 
-To use the AWS CLI with AWS Batch, install the latest AWS CLI, version\. For information about installing the AWS CLI or upgrading it to the latest version, see [Installing the AWS Command Line Interface](http://docs.aws.amazon.com/cli/latest/userguide/installing.html) in the *AWS Command Line Interface User Guide*\.
\ No newline at end of file
+To use the AWS CLI with AWS Batch, install the latest AWS CLI, version\. For information about installing the AWS CLI or upgrading it to the latest version, see [Installing the AWS Command Line Interface](http://docs.aws.amazon.com/cli/latest/userguide/installing.html) in the *AWS Command Line Interface User Guide*\.
```
Comment (nrdlngr) [inline doc_source/get-set-up-for-aws-batch.md]: Please remove the comma in "AWS CLI, version".

**Review by nrdlngr — CHANGES_REQUESTED** [review]
Sorry I misread some of your suggestions on first review. The GitHub diff wasn't flagging some of them so I missed them. I can merge this after you address my comments. Thanks!

**Comment by landermkerbey** [comment]
Yeah, I noticed that Github was displaying the diffs oddly--for example, in that final paragraph it showed a diff even though I didn't change anything, and it showed the whole paragraph as a diff for that 'or and' thing. I went ahead and made changes reflective of your comments, let me know what you think.

By the way, thanks for being so quick on the draw with the reviews!

File: doc_source/get-set-up-for-aws-batch.md [hunk 4]
```diff
@@ -95,7 +95,7 @@ For more information about IAM, see the [AWS Identity and Access Management User
 
 ## Create IAM Roles for your Compute Environments and Container Instances<a name="create-an-iam-role"></a>
 
-Your AWS Batch compute environments and container instances require AWS account credentials to make calls to other AWS APIs on your behalf\. You must create IAM roles that provides these credentials to your compute environments and container instances and then associate that role with your compute environments\.
+Your AWS Batch compute environments and container instances require AWS account credentials to make calls to other AWS APIs on your behalf\. You must create an IAM role that provides these credentials to your compute environments and container instances, then associate that roles with your compute environments\.
```
Comment (nrdlngr) [inline doc_source/get-set-up-for-aws-batch.md]: Should be "associate that role with".

**Comment by landermkerbey** [comment]
Woops. Fixed.

**Comment by nrdlngr** [comment]
Sorry I let this sit idle for so long. Thanks for your contribution!

**Pull request merged**